app/user-setup/index.tsx

```diff
@@ -144,11 +140,14 @@ function UserSetupPage() {
                     control={control}
                     name="name"
                     render={({ field: { onChange, value } }) => (
-                        <TextInput
-                            className="border-hairline h-14 rounded-md border-black px-3 py-1.5 text-lg text-gray-900"
+                        <Input
                             placeholder="Enter Display Name"
                             onChangeText={onChange}
                             value={value}
+                            textStyle={{
+                                marginHorizontal: 10,
+                            }}
```

Comment:
convert to `textClassName`

```tsx
textClassname="mx-something"
```

Reply:
also bai for all inputs gane, ma use ni nimo nga property nila ay

```tsx
status={
    errors.name
        ? "danger"
        : "basic"
}
caption={errors.name?.message}
```

this one kay para same jud for errors sad, pa convert ko bai sa tanan `{errors.x &&...` stuff here sa component with that caption thingy

pero for other error messages nga di ga use ug Input caption, retain lang